stories/panel.stories.ts

```diff
@@ -151,11 +151,13 @@ storiesOf('Basics/Panel', module)
     withHits(
       ({ search, container }) => {
         const breadcrumbInPanel = panel<typeof breadcrumb>({
-          collapsed({ widgetRenderState }) {
-            return widgetRenderState.canRefine;
+          collapsed({ canRefine }) {
```

Comment:
This usage is much better and eventually what we want to enable in next major version so if there can't be collision with the IS instance (which is also passed and spread) that's great!

Reply:
Yeah, I didn't realize there cannot be collision. This looks so much better.

Reply:
as far as I can tell there's no collision here, so this doesn't even need a major :)

Reply:
Yeah, it makes sense there is no collision :)